README.md

```diff
@@ -40,6 +40,8 @@ for example usage.
 
 ## Development
 
+Ensure you are using Node v8.
```

Comment:
Good find. I just submitted this PR which will avoid the need to stick to Node 8:
https://github.com/Jigsaw-Code/outline-shadowsocksconfig/pull/8

Reply:
@joeytwiddle With https://github.com/Jigsaw-Code/outline-shadowsocksconfig/pull/8 merged you should be able to revert this.

Reply:
Done